src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+
+const table = document.querySelector('.field');
+const rows = table.querySelectorAll('tr');
```

Comment:
The `rows` variable is initialized here but never updated. This could lead to errors when adding rows, as the length of `rows` will not reflect the current state of the table. Consider updating `rows` dynamically or using `table.querySelectorAll('tr')` directly when needed.